brokerpaks/empty/empty.yml

```diff
@@ -0,0 +1,101 @@
+version: 1
```

Comment:
Are we going to permanently leave this brokerpak in here?

Reply:
Nope, this is a test one to try to get the broker deploying. Real paks soon to come!